cluster-autoscaler/core/scale_down_test.go

```diff
@@ -1209,6 +1209,15 @@ func getStringFromChan(c chan string) string {
 	}
 }
 
+func getGroupSizeChangeFromChan(c chan groupSizeChange) *groupSizeChange {
```

Comment:
As far as I can see you only use it in scale_up_test.go, why do you define it here?